test/fixtures/bundle.js

```diff
@@ -1,7 +1,34 @@
 import { insertModule, createModuleUrl, resolveImportSpecifier } from "/bloom.js";
+insertModule("test/fixtures/src/b.js",createModuleUrl(`const b = 're-export';
+
+export { b };
+`));
+insertModule("test/fixtures/src/e.js",createModuleUrl(`const e = 're-export e-aggregate';
+
+export { e };
+`));
+insertModule("test/fixtures/src/d.js",createModuleUrl(`const d = 're-export default';
+
+export default d;
+`));
+insertModule("test/fixtures/src/a-default.js",createModuleUrl(`const A = () => {};
+
+export { A };
+`));
 insertModule("test/fixtures/src/a.js",createModuleUrl(`export const a = 'a value';
-export const other = 'other value';`));
-insertModule("test/fixtures/src/main.js",createModuleUrl(`import { a, other } from '${resolveImportSpecifier("test/fixtures/src/a.js")}';
 
-console.log('hi there', a, other);`));
+export { b } from './b.js';
```

Comment:
I tested the output bundle with the following code (we need to create automated tests, but here's how i do it manually for now):


```sh
cd test/fixtures
http-server .
open http://localhost:8080
```

I got the following error:

> Failed to resolve module specifier "./b.js". Invalid relative url or base scheme isn't hierarchical

The reason is that the module specifiers for exports are not being replaced with a `resolveImportSpecifier()` call, like they are for imports. Compare the line here to line 30